src/test/kotlin/it/pagopa/ecommerce/payment/requests/controllers/PaymentRequestsControllerTests.kt

```diff
@@ -108,7 +108,7 @@ class PaymentRequestsControllerTests {
   @Test
   fun `should return response entity with party configuration fault`() = runTest {
     val rptId = "77777777777302000100000009424"
-    val faultBean = faultBeanWithCode(PartyConfigurationFaultDto.PPT_DOMINIO_DISABILITATO.value)
+    val faultBean = faultBeanWithCode(PartyConfigurationFaultDto.PAA_ID_DOMINIO_ERRATO.value)
```

Comment:
Please parameterize all those tests against all values in enumeration in order to test all associations and make it easy to discover some fault code that has wrongly associated to different fault code categories. Something as done [here](https://github.com/pagopa/pagopa-ecommerce-payment-requests-service/pull/96/files#diff-c815c66ada0267de914e6deddfca6da3b905df0cc6de78866b254a39efe322f8R133-R134)

Reply:
done, sounds good :-)